src/var/isWindow.js

```diff
@@ -0,0 +1,8 @@
+define( function() {
+	"use strict";
+
+	return function isWindow( obj ) {
+		return obj !== null && typeof obj !== "undefined" && obj === obj.window;
```

Comment:
I believe I added it because a test was breaking. Let me remove it and check if it still breaks.